src/OddJobs/Cli.hs

```diff
@@ -155,35 +160,23 @@ defaultWebUI UIStartArgs{..} uicfg@UIConfig{..} = do
       Warp.run uistartPort $ Servant.serveWithContext api ctx app
 
 {-| Used by 'defaultMain' if 'Stop' command is issued via the CLI. Sends a
-@SIGINT@ signal to the process indicated by 'shutPidFile'. Waits for a maximum
+@SIGTERM@ signal to the process indicated by 'shutPidFile'. Waits for a maximum
```

Comment:
By default, `SIGTERM` will kill the process dead with no graceful shutdown because the Haskell runtime doesn't install a handler for `SIGTERM`, only `SIGINT`. I think it would be helpful to provide a handler here, so that the user of this library doesn't have to install their own, and because handling `SIGTERM` is what daemons are supposed to do.

I've placed some code you can adapt in a [gist](https://gist.github.com/neilmayhew/56d7a4160d29054b95ce994b74bdca1e).

Reply:
Neil, I have duly obliged and shamelessly lifted your code into this module :wink:. I used `withGracefulTermination_` to wrap the lambda on the following line where the daemon is spawned: https://github.com/saurabhnanda/odd-jobs/blob/50d27b46cd934657b7befde5841bd37f18e540b0/src/OddJobs/Cli.hs#L150-L155

Reply:
For some reason, GitHub isn't giving me the option to resolve this conversation. However, I do consider it resolved.

Thanks for making this change.

Reply:
Thanks so much for the approval, Neil. It looks like I'm able to resolve the conversation on my end.